### ISSUE TITLE
busted --lpath isn't functioning

### ISSUE THREAD

**Issue by TheCycoONE** [Olivine-Labs/busted#327]
As can be seen in our pull request https://github.com/CorsixTH/CorsixTH/pull/661, busted --lpath isn't functioning installed in a clean (TravisCI) environment with luarocks.

A workaround is to prepend the path to LUA_PATH manually.

working:
luafilesystem 1.6.2-2
busted 2.0.rc3-0

non-working:
luafilesystem 1.6.3-1
busted 2.0.rc5-0

**Comment by o-lim** [comment]
Fixed in PR #329 .  However, it was technically not working in busted 2.0.rc3-0 either.  I looked more closely at your repo, and found that you have a `.busted` file with a typo (backslash instead of forward slash).
```lua
return {
  default = {
    lpath = "../Lua\?.lua",
  },
}
```
when it should be
```lua
return {
  default = {
    lpath = "../Lua/?.lua",
  },
}
```
The error was due to the way the `.busted` file was being loaded.  The `--lpath` command-line argument was being overridden with the value in the `.busted` file, when it should have been the other way around.  The command-line arguments should override the `.busted` file.

If you fix your `.busted` file, you shouldn't need to use `--lpath` in your rockspec.
